cmake/Dependencies.cmake

```diff
@@ -449,6 +449,98 @@ if(USE_XNNPACK)
   list(APPEND Caffe2_DEPENDENCY_LIBS XNNPACK)
 endif()
 
+if(USE_VULKAN)
+  # Commenting for prototype, using vulkan_wrapper directly for simplicity of prototype
+  # add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/../third_party/Serenity)
+  # list(APPEND Caffe2_DEPENDENCY_LIBS Orion)
+
+  if(NOT ANDROID)
```

Comment:
How hard is it to remove this restriction?

Reply:
It should not be hard - at the moment it tries to find vulkan, vulkan_wrapper, shaderc from android ndk path.
We just need to support case  to get vulkan and vulkan_wrapper from VULKAN_SDK, shaderc clonning from github and building. In Ashkans library there is already setup for VULKAN_SDK and separate shaderc case.